src/renderer/store/modules/subscription-cache.js

```diff
@@ -0,0 +1,277 @@
+import {
+  DBSubscriptionCacheHandlers,
+} from '../../../datastores/handlers/index'
+
+const state = {
+  videoCache: {},
+  liveCache: {},
+  shortsCache: {},
+  postsCache: {},
+
+  subscriptionCacheReady: false,
+}
+
+const getters = {
+  getSubscriptionCacheReady: (state) => state.subscriptionCacheReady,
+
+  getVideoCache: (state) => {
+    return state.videoCache
+  },
+
+  getVideoCacheByChannel: (state) => (channelId) => {
+    return state.videoCache[channelId]
+  },
+
+  getShortsCache: (state) => {
+    return state.shortsCache
+  },
+
+  getShortsCacheByChannel: (state) => (channelId) => {
+    return state.shortsCache[channelId]
+  },
+
+  getLiveCache: (state) => {
+    return state.liveCache
+  },
+
+  getLiveCacheByChannel: (state) => (channelId) => {
+    return state.liveCache[channelId]
+  },
+
+  getPostsCache: (state) => {
+    return state.postsCache
+  },
+
+  getPostsCacheByChannel: (state) => (channelId) => {
+    return state.postsCache[channelId]
+  },
+}
+
+const actions = {
+  async grabAllSubscriptions({ commit, dispatch, rootGetters }) {
+    try {
+      const payload = await DBSubscriptionCacheHandlers.find()
+
+      const videos = {}
+      const liveStreams = {}
+      const shorts = {}
+      const communityPosts = {}
+
+      const toBeRemovedChannelIds = []
+      const subscribedChannelIdSet = rootGetters.getSubscribedChannelIdSet
+
+      for (const dataEntry of payload) {
+        const channelId = dataEntry._id
+        if (!subscribedChannelIdSet.has(channelId)) {
+          // Clean up cache data for unsubscribed channels
+          toBeRemovedChannelIds.push(channelId)
+          // No need to load data for unsubscribed channels
+          continue
+        }
+
+        let hasData = false
+
+        if (Array.isArray(dataEntry.videos)) {
+          videos[channelId] = { videos: dataEntry.videos, timestamp: dataEntry.videosTimestamp }
+          hasData = true
+        }
+        if (Array.isArray(dataEntry.liveStreams)) {
+          liveStreams[channelId] = dataEntry.liveStreams
```

Comment:
Looks like this line is probably left over from something, as it doesn't exists for the other types.